internal/location/schema.go

```diff
@@ -1,62 +1,38 @@
 package location
 
 import (
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
 )
 
-// Schema returns the default Schema which should be used for Location fields
-// where these are Required and Cannot be Changed
 func Schema() *pluginsdk.Schema {
-	return &pluginsdk.Schema{
-		Type:             pluginsdk.TypeString,
-		Required:         true,
-		ForceNew:         true,
-		ValidateFunc:     EnhancedValidate,
-		StateFunc:        StateFunc,
-		DiffSuppressFunc: DiffSuppressFunc,
-	}
+	return commonschema.LocationSchema()
```

Comment:
does it make sense to have schema on the function name? other functions in there don't?

Reply:
(as discussed) will pick this off OOB in the next helpers release